ghost/milestones/lib/InMemoryMilestoneRepository.js

```diff
@@ -27,6 +38,10 @@ module.exports = class InMemoryMilestoneRepository {
         } else {
             this.#store.push(milestone);
             this.#ids[milestone.id.toHexString()] = true;
+
+            for (const event of milestone.events) {
```

Comment:
This should run outside of the `if/else` block - because we want to emit events for all saves.

Obviously ATM we only have an event when it's saving a new milestone - but I think for correctness it's better to move it :)